pkg/test/utils.go

```diff
@@ -143,6 +143,7 @@ func RunCommand(command string) string {
 }
 
 func DeserializeJSONToMap(t *testing.T, in string) config.GenericMap {
+	t.Helper()
```

Comment:
What is the practical result of declaring this function as a Helper function?

Reply:
@KalmanMeth 
By marking a function as helper, when a test fails in the helper function, the line number reported will be of the caller function rather than inside the test helper function.

Consider running the following test:

```golang
package b

import (
	"testing"
)

func Test(t *testing.T) {
	helperMarked(t, "hello", "hi") // line 8
	helperUnmarked(t, "hello", "hi")
}

func helperMarked(t *testing.T, expected, actual string) {
	t.Helper()
	if expected != actual {
		t.Errorf("expected (%v) != actual(%v)", expected, actual)
	}
}
func helperUnmarked(t *testing.T, expected, actual string) {
	if expected != actual {
		t.Errorf("expected (%v) != actual(%v)", expected, actual) // line 20
	}
}
```

The result is:
```
=== RUN   Test
    b_test.go:8: expected (hello) != actual(hi)
    b_test.go:20: expected (hello) != actual(hi)
--- FAIL: Test (0.00s)

FAIL
```

Reply:
nice, I didn't know that